src/App/routes/index.js

```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express';
+import blogs from './blog';
+import users from './user';
+import auth from './auth';
+
+const router = Router();
+
+router.use('/api/v1/techblogs/blogs', blogs);
```

Comment:
Can you move the routes prefix in the `src/index.js` so as to remain with a shorter route here please? This is to remove the repetitions.
```suggestion
router.use('/blogs', blogs);
```